__tests__/testCases/uroky202502Input.ts

```diff
@@ -0,0 +1,19 @@
+import { TAX_YEAR, UROKY_POCET_ROKOV } from '../../src/lib/calculation'
+import { E2eTestUserInput } from '../../src/types/E2eTestUserInput'
+import { case202302Input } from './case202302Input'
+import { urokyEligibleInput } from './urokyEligibleInput'
+
+export const uroky202502Input: E2eTestUserInput = {
+  ...case202302Input,
+  ...urokyEligibleInput,
+  r035_uplatnuje_uroky: false,
```

Comment:
```suggestion
  r035_uplatnuje_uroky: true,
```

tu chceme aby v otazke klikol ze chcem uplatnit uroky ale aby az po vyplneni dostal info ze nema narok a teda do `executeCase` musime doplnit flag `uroky_nesplna_datumy` a tam bude if ktory skontroluje ze je tam text ktory mu hovori ze nesplna a klikne dalej.